### ISSUE TITLE
Could you highlight difference with remember cursor position?

### ISSUE THREAD

**Issue by MaxLap** [ludovicchabant/obsidian-remember-file-state#10]
Hello,

This plugin and [remember cursor position](https://github.com/dy-sh/obsidian-remember-cursor-position) appear to basically be the same thing, except one works when closing Obsidian and the other doesn't. 

When searching for "remember", both show up, causing a bit of confusion. Reading the description of this project, it appears that this does a subset of what remember cursor position does. i.e.: The "cursor position" one says "cursor position, scrolling and text selection", while "file state" says "cursor position, scrolling position, and so on". It's unclear what is included in "so on", adding to the confusion.

I think it would be helpful for users if you expanded a little on what this does in the readme and maybe highlighted the difference between compared to remember-cursor-position.

Regards

**Comment by ludovicchabant** [comment]
Hi! Thanks, for the heads-up. I actually was totally unaware of the other plugin when I wrote this one... so I guess they do the same thing!

At first glance, one thing that other plugin does is setup a regular timer that triggers editor states checks/saves at regular intervals. I took some extra effort to avoid this and really only do things on file open/close, in order to not slow down the user experience. The other plugin also has some short customizable timer for when to restore the state after opening a file. I know the sort of problem it's trying to avoid by using a timer, but once again I made some effort to just make it work and not offer some fiddly setting like this to the user.

On the other hand, the other plugin seems to save states between editing sessions, which my plugin doesn't do (yet).

Of course I haven't tested the other plugin so I could be wrong!
